constraints.txt

```diff
@@ -47,33 +48,28 @@ packaging==19.0
 parsimonious==0.8.1
 pendulum==2.0.4
 pep8-naming==0.8.2
-pip==19.1.1
```

Comment:
please keep the constraints for at least pip, wheel and setuptools in there. You can run `pip freeze --all` to also show them.
Unless there is a good reason to remove the others, I would also keep them in there.